thunder/executors/transformer_engineex.py

```diff
@@ -544,7 +544,7 @@ def te_sync_fp8_meta_bwd_impl():
 )
 
 
-def _transformer_engine_bwd_fp8_meta_sync(fw_extrace, bw_extrace):
+def _transformer_engine_bwd_fp8_meta_sync(unused_fw_extrace, bw_extrace):
```

Comment:
Usually, "_" is used for unused variables, and IDEs are able to recognize it correctly with proper syntax highlighting.